keycloak_api_client/api_client.py

```diff
@@ -326,3 +329,29 @@ def count_users(
                 f"(msg: {response.json()})"
             )
         return response.json()
+
+    def reset_password(
+        self,
+        keycloak_id: UUID,
+        new_password: str,
+        temporary: Optional[bool] = False
```

Comment:
`Optional` is invalid unless you accept `temporary=None`. I'm not sure if
```json
{"type": "password", "new_password": "...", "temporary": null}
```
is an accepted input.